library/kotlin/src/io/envoyproxy/envoymobile/RequestHeadersBuilder.kt

```diff
@@ -35,6 +35,26 @@ class RequestHeadersBuilder : HeadersBuilder {
    */
   internal constructor(headers: MutableMap<String, MutableList<String>>) : super(headers)
 
+  override fun add(name: String, value: String): RequestHeadersBuilder {
+    super.add(name, value)
+    return this
+  }
+
+  override fun set(name: String, value: MutableList<String>): RequestHeadersBuilder {
+    super.set(name, value)
+    return this
+  }
+
+  override fun remove(name: String): RequestHeadersBuilder {
+    super.remove(name)
+    return this
+  }
+
+  override fun internalSet(name: String, value: MutableList<String>): RequestHeadersBuilder {
+    super.internalSet(name, value)
+    return this
+  }
+
```

Comment:
These are necessary in order to return a `RequestHeadersBuilder` type rather than a `HeadersBuilder`